Framework/Algorithms/src/GeneratePythonScript.cpp

```diff
@@ -44,9 +45,14 @@ void GeneratePythonScript::init() {
                   "Unroll all algorithms to show just their child algorithms.",
                   Direction::Input);
 
+  declareProperty("IgnoreGroups", false,
```

Comment:
New properties should be declared at the end of `init()`. Adding properties in the middle will break Python scripts that don't use named arguments but rely on the order of the properties.

Reply:
Resolved this now.